fluidcoins/addresses.go

```diff
@@ -0,0 +1,194 @@
+package fluidcoins
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/fluidcoins/client-go/util"
+	"github.com/google/uuid"
+)
+
+type AddressesService service
+
+type Address struct {
+	ID         uuid.UUID `json:"id"`
+	Address    string    `json:"address"`
+	Coin       Coin      `json:"coin"`
+	Domain     string    `json:"domain"`
+	MerchantID string    `json:"merchant_id"`
+	// Refernece is the unique ID for this address. You can use this id to fetch the address again
+	Reference string          `json:"reference"`
+	Metadata  AddressMetadata `json:"metadata"`
+
+	CreatedAt time.Time `json:"created_at"`
+	UpdatedAt time.Time `json:"updated_at"`
+}
+
+type Coin struct {
+	ID   uuid.UUID `json:"id"`
+	Code string    `json:"code"`
+
+	HumanReadableName string `json:"human_readable_name"`
+	// This is a base64 encoded image not an actual url resource
```

Comment:
```suggestion
        // This is a url link ( hosted by us ) to the logo of the coin.
```

Reply:
It is not a base64 encoded image

Reply:
ok, you should also perhaps update from your end (https://developers.fluidcoins.com/reference/get_address)
<img width="952" alt="Screenshot 2022-03-04 at 02 23 31" src="https://user-images.githubusercontent.com/29808443/156681140-d39d4318-6a0a-432c-be3b-bfdbbb95b5a5.png">

Its the same for all mentions of `Image`

Reply:
updated

Reply:
Ha damn, will update the docs